src/common/mustacheHelper.ts

```diff
@@ -320,6 +320,7 @@ export class MustacheHelper {
     mapSeverityCount(imageScanExecutionInfo:any):severityCount | {} {
         if (imageScanExecutionInfo && imageScanExecutionInfo.severityCount){
             return {
+                critical: imageScanExecutionInfo.severityCount.critical,
                 high: imageScanExecutionInfo.severityCount.high,
                 moderate: imageScanExecutionInfo.severityCount.moderate,
```

Comment:
We should update Moderate to medium. As Moderate has been removed from cve store now. Aslo should add unknown